src/entities/elastic-storage/tx-data.js

```diff
@@ -34,7 +34,7 @@ class TxData extends ElasticData {
       outputs: [
         {
           address: utxo.receiver,
-          amount: utxo.value,
+          value: utxo.amount,
```

Comment:
Why not maintain the same naming? Amount: utxo.amount

Reply:
Non-critical, but I will take a not. Since fix involves a separate file with the type - let's make it a separate PR